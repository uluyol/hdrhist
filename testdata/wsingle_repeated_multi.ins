0
1
123
123
145
145
555
666
666
1400000
540000
9
1
---
