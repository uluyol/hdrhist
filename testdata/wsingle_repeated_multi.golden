0.000,0.001,1.401,HISTFAAAADR42pNpmSzMwMAgxgABzFCaEUzK//9v/wEqwsTL9JCRRZNlIxvTbUaW1mmMTO9FmADoYgmN
